public/blog/_data.json

```diff
@@ -183,5 +183,15 @@
     "title": "Intel XDK: Develop Once, Deploy Everywhere",
     "date": "2014-03-03T11:00",
     "author": "Fredrick Odhiambo"
+  },
+  "chinese-web-release": {
+    "title": "Crosswalk Website has been released!",
+    "date": "2016-05-20T12:00",
+    "author": "Lisha"
```

Comment:
First and Last name has been our standard process.  There may be dozens of Lisha's :)